1-js/08-prototypes/02-function-prototype/1-changing-prototype/solution.md

```diff
@@ -1,20 +1,20 @@
 
-Answers:
+Respostas:
 
 1. `true`. 
 
-    The assignment to `Rabbit.prototype` sets up `[[Prototype]]` for new objects, but it does not affect the existing ones. 
+    A atribuição a `Rabbit.prototype` configura o `[[Prototype]]` para novos objetos, mas não afeta os objetos que já existem.
 
 2. `false`. 
 
-    Objects are assigned by reference. The object from `Rabbit.prototype` is not duplicated, it's still a single object referenced both by `Rabbit.prototype` and by the `[[Prototype]]` of `rabbit`. 
+    Objetos são atrbuídos por referência. O objeto do `Rabbit.prototype` não é duplicado, ele continua sendo um único objeto referenciado por `Rabbit.prototype` e pelo `[[Prototype]]` do `rabbit`. 
```

Comment:
> Grammatical Error

- Change `atrbuídos` to `atribuídos`

<img width="270" alt="image" src="https://user-images.githubusercontent.com/35238611/177467856-d8d67272-cd32-40ad-b8a8-923f0a9ed34a.png">